package/gluon-config-mode-core/luasrc/lib/gluon/config-mode/model/gluon-config-mode/wizard.lua

```diff
@@ -18,21 +9,10 @@ local s = f:section(Section)
 s.template = "wizard/welcome"
 s.package = "gluon-config-mode-core"
 
-local commit = {'gluon-setup-mode'}
```

Comment:
We still need to save `gluon-setup-mode`. Currently, everything else is saved w/o problems but we're stuck in setup mode.